backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -55,8 +55,7 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
     case _ => false
   }
 
-  // This seems dangerous as any exception that is not one of the above (fatal) will trigger infinite retries !
-  def isTransient(throwable: Throwable): Boolean = !isFatal(throwable)
+  def isTransient(throwable: Throwable): Boolean = false
```

Comment:
Can we get rid of this method (and `isFatal`) and only keep the `false` paths elsewhere?

Reply:
not sure what you mean ? `isFatal` and `isTransient` are needed by the `withRetry` method

Reply:
Ignore me. Since it's now hard-coded to false, I thought we could remove the method call... but I guess it must be overridden so we *do* have to keep it!